src/app/features/courses/courses.module.ts

```diff
@@ -1,11 +1,10 @@
 import { NgModule } from '@angular/core';
 import { SharedModule } from 'src/app/shared/shared.module';
 import { CourseCardComponent } from './courseCard.component';
-import { CoursesComponent } from './courses.component';
 
 @NgModule({
     imports: [SharedModule],
-    declarations: [CoursesComponent, CourseCardComponent],
-    exports: [CoursesComponent, CourseCardComponent],
+    declarations: [CourseCardComponent],
+    exports: [CourseCardComponent],
```

Comment:
lets use ChangeDetection Onpush for all the components

Reply:
done